src/state_transition/test_utils/generate_state.zig

```diff
@@ -15,6 +17,7 @@ const PubkeyIndexMap = state_transition.PubkeyIndexMap(ValidatorIndex);
 const Index2PubkeyCache = state_transition.Index2PubkeyCache;
 const syncPubkeys = state_transition.syncPubkeys;
 const interopPubkeysCached = @import("./interop_pubkeys.zig").interopPubkeysCached;
+const active_chain_config = if (preset.preset == Preset.mainnet) mainnet_chain_config else minimal_chain_config;
```

Comment:
we're already depending on `build_options`, can we just do this maybe?

```suggestion
const preset_str = @import("build_options").preset;

const active_chain_config = if (std.mem.eql(u8, preset_str, "mainnet")) mainnet_chain_config else minimal_chain_config;
```

if i'm not wrong, the `preset` field is only used to check this, so we can just get rid of that if we do this possibly